### ISSUE TITLE
[Bug] Adding user to organization does not allow him to view dashboards

### ISSUE THREAD

**Issue by tomgs** [grafana/grafana#18635]
**What happened:** When inviting a user to join my organisation, he clicks the invite link, creates a user name and a password, and then logs in. He cannot see any of the dashboards. 

Upon further investigation, it seems that the database row is entered incorrectly into `public.user` - he is part of the organisation with `org_id` `-1`, instead of the organisation with `org_id` `1` - the organisation he was invited to. This is probably why he can login (he has an entry in the user table) but cannot see the dashboards (he is assigned to the wrong organisation - namely a non-existing one).

**What you expected to happen:** When I invite a user to my organisation, I presume he is added to the organisation. That is not the case here.

**How to reproduce it (as minimally and precisely as possible):** When using Grafana-in-docker and an external Postgres container as a database, try and add a user to your organisation (I'm an admin, so probably best to repro as admin). Then log into the database container and the `grafana` database and run:

```
grafana=# select * from public.user;
 id | version |             login             |             email             |         name         |                                               password                                               |    salt    |   rands    | company | org_id | is_admin | email_verified | theme |       created       |       updated       | help_flags1 |    last_seen_at     | is_disabled 
----+---------+-------------------------------+-------------------------------+----------------------+------------------------------------------------------------------------------------------------------+------------+------------+---------+--------+----------+----------------+-------+---------------------+---------------------+-------------+---------------------+-------------
.....
 {id-given-to-user} |       0 | {username-of-user}                  | {email-of-user}       | {fullname-of-user  | {password} | {salt} | {rands} |         |     -1 | f        | f              |       | 2019-08-20 06:54:56 | 2019-08-20 06:54:56 |           0 | 2019-08-20 07:07:44 | f
```

**Note:** This can easily be fixed:
1. In `public.user` - edit the `org_id` to be the correct one.
2. In `org_user` - add an entry for that user with the correct `user_id`, `org_id` and `role`.
But it doesn't make sense to do it manually for each user. 

**Anything else we need to know?:** n/a

**Environment:**

Grafana version: 6.3.2
Data source type & version: Postgres
OS Grafana is installed on: Inside a Docker container with the official Grafana image, and Postgres is on a different container. This is a `docker-compose` multi-container app.
User OS & Browser: Irrelevant
Grafana plugins: None
Others: N/A

**Comment by torkelo** [comment]
What are your grafana.ini settings?

**Comment by torkelo** [comment]
Very strange, I tested to invite just now and used the signup page to complete the profess and was automatically added to org_id 1 (default org).

**Comment by tomgs** [comment]
So, it really seems like it's a problem on our end:
We recently migrated from sqlite3 to postgres as the backend for grafana. 
I didn't see any **exact** steps to follow for that migration (maybe it's in the docs and I missed it? Would appreciate a link if it's there) so I basically did it manually - set up a fresh grafana instance using postgres, extracted a dump out of sqlite3 and hand-written the queries to populate all the tables in postgres.

This seems to have cause a sequence problem, which we solved by updating all of them manually. This can be closed, but it would be nice if there were proper steps for migrating from grafna 5.x to 6.x and from sqlite3 to postgres, at the same breath.
